package/addons-update.sh

```diff
@@ -67,21 +67,20 @@ else
   INFLUXDB_VOLUME="hostPath:\n          path: $INFLUXDB_HOST_PATH"
 fi
 
+# Addons Images
+# If any of these versions are updated, please also update them in
+# addon-templates/README.md
 
 ADDONS_DIR=/etc/kubernetes/addons
-# image list generate from rancher/types
-# https://github.com/rancher/types/blob/master/apis/management.cattle.io/v3/k8s_defaults.go
-source addons_image_list
-# rancher/types has the images with registry namepsace. Since we have that configurable here, we need to remove it:
-TILLER_IMAGE=${TILLER_IMAGE#*/}
-KUBEDNS_IMAGE=${KUBEDNS_IMAGE#*/}
-DNSMASQ_IMAGE=${DNSMASQ_IMAGE#*/}
-GRAFANA_IMAGE=${GRAFANA_IMAGE#*/}
-HEAPSTER_IMAGE=${HEAPSTER_IMAGE#*/}
-INFLUXDB_IMAGE=${INFLUXDB_IMAGE#*/}
-DASHBOARD_IMAGE=${DASHBOARD_IMAGE#*/}
-DNS_SIDECAR_IMAGE=${DNS_SIDECAR_IMAGE#*/}
 
```

Comment:
It seems like we're missing removing some line? https://github.com/rancher/kubernetes-package/pull/138/commits/8998f276c8d8bae1b867439bbb67951909fdc50e#diff-347380d9bf27d44c881730a8a2906d3cR72

Reply:
Line you referenced is already removed.